sample_annotator/clients/submission_portal_client.py

```diff
@@ -0,0 +1,189 @@
+import nmdc_schema.nmdc as nmdc
+import linkml
+import linkml_runtime.dumpers.json_dumper as dumper
```

Comment:
`from linkml_runtime.dumpers import json_dumper`

Reply:
Implemented this change.